Set Python runtime explicitly rather than using the default one.

--- iso-codes-3.21/iso2pot.py
+++ iso-codes-3.21/iso2pot.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2.7
 # coding: utf-8
 #
 # Read iso-codes data file and output a .pot file
